test/test-utils/shim.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * React requires the requestAnimationFrame to be defined for tests.
+ */
+// Prevent typescript from giving error.
+const globalAny: any = global;
+
+globalAny.requestAnimationFrame = callback => {
+  setTimeout(callback, 0);
```

Comment:
This is the long way to type it, you could just do `(global as any).requestAnimationFrame`